qiskit_quantuminspire/qi_jobs.py

```diff
@@ -34,7 +59,7 @@ def __init__(
         self,
         run_input: Union[QuantumCircuit, List[QuantumCircuit]],
         backend: Union[Backend, None],
-        job_id: str,
+        job_id: Optional[int] = None,
```

Comment:
Should a user still be able to submit a job if job_id is not None? Should we somehow verify the job_id actually exists?

Reply:
Upon closer look and discussion with Nischal, we do not actually need this. This id that goes into the Job constructor is an arbitrary, user provided identifier. We will get the batch_job_id after submission and do not need to provide it ourselves.